src/main/java/cn/nukkit/item/Item.java

```diff
@@ -73,17 +74,10 @@ public Item(int id, Integer meta, int count, String name) {
         }*/
     }
 
-    public boolean hasMeta() {
-        return hasMeta;
-    }
-
-    public boolean canBeActivated() {
-        return false;
-    }
-
     public static void init() {
         if (list == null) {
             list = new Class[65535];
+            customblocklist = new Class[65535];
```

Comment:
is it not possible to make this smaller somehow? use a trove/fastutil map or something, both of these arrays are 512k (on 64-bit JVMs), which is quite a substantial amount when you think about it, especially since in most scenarios, most of it won't be used. (yes, this is also a problem with the `list` field, but that's already kinda grandfathered in).

side note: shouldn't the length be 65536 here? i'm assuming you just copied the value from `list`, but it seems wrong.